Kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -58,6 +58,10 @@ data:
         # - prometheus.io/port: If port is not 9102 use this annotation
         monitor_kubernetes_pods = false
 
+        ## Restricts Kubernetes monitoring to namespaces for pods. This will take effect when monitor_kubernetes_pods is set to true
+        ##   ex: monitor_kubernetes_pods_namespaces = "default1, default2, default3"
+        # monitor_kubernetes_pods_namespaces = ""
```

Comment:
make it an array, and also empty arrays is a valid scenario for all array inputs.